app/src/pages/inside/profilePage/localizationBlock/localizationBlock.jsx

```diff
@@ -121,6 +132,8 @@ export class LocalizationBlock extends Component {
   };
 
   onChangeLanguage = (lang) => {
+    console.log('change lang');
```

Comment:
Looks like you've forgot to remove `console.log`

Reply:
fixed